### DISCUSSION TITLE
Gitbook not loading after login.

### DISCUSSION THREAD

**Discussion by shnmrc** [GitbookIO/community#440] [0. General bugs and questions] [answered]
Hello! Experiencing issues with using gitbook.
Once i do signin, i'm getting message that Connection lost. Tried with VPN and without, nothing helps. What's can be wrong?
![Снимок экрана 2024-01-03 в 18 25 47](https://github.com/GitbookIO/community/assets/45079779/b127355b-160e-4b3d-b4a4-7a6c73aded94)

**Comment by shnmrc** [comment, marked as answer]
Solved...  [CORS plugin](https://webextension.org/listing/access-control.html) for chrome was blocking firestore. Weird!

**Reply by aikoii** [reply]
👋 @shnmrc! Glad to hear you've resolved your issue. And yes, we've seen cases, where CORS settings disrupt access to GitBook.
